interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
Samir, please use this section as an opportunity to prepare for interviews. How you represent yourself during an interview is what gets you hired! Please go through this. Research and find something you did not know about it. And share an experience you had with that topic.